lib/core_types/core_types.dart

```diff
@@ -1,8 +1,8 @@
+export 'bool.dart';
```

Comment:
This change is not actually acceptable changing the order of exporting does not matter.

Reply:
I see. It's the only one not in order. I thought it should lol

Reply:
Undo the changes